web/src/components/FiltersForm.res

```diff
@@ -1,8 +1,58 @@
 open Prelude
 
+module RelativeDate = {
```

Comment:
It is working as expected except that I will need to import that module in common.js for queryBackend to use the right date for gte just after the DOM is created.